requirements.txt

```diff
@@ -26,3 +26,6 @@ sentencepiece==0.1.95
 # for code format
 black==21.5b2
 pre-commit==2.13.0
+
+#for number-to-word
+inflect==5.3.0
```

Comment:
@kaustubhdhole pending -- shall we directly change the requirement.txt, or would it make more sense if we customize readmes for different transformations?

Reply:
1. Sure! I will move it to transformations/number-to-word/requirements.txt 
2. I will look into it